src/lucky/web_socket.cr

```diff
@@ -0,0 +1,8 @@
+module Lucky
+  class WebSocket < HTTP::WebSocketHandler
+    getter proc
+
+    def initialize(@action : Lucky::Action.class, &@proc : HTTP::WebSocket, HTTP::Server::Context -> Void)
```

Comment:
This `@action` isn't used at all, but if you have several different websocket routes, this at least lets us differentiate between them if we need.